README.md

```diff
@@ -23,6 +23,16 @@ rocks trees.
 * [Installation instructions for macOS](http://luarocks.org/en/Installation_instructions_for_macOS)
 * [Installation instructions for Windows](http://luarocks.org/en/Installation_instructions_for_Windows)
 
+## Contributing
+
+* Create a new branch with a meaningful name `git checkout -b branch_name`.<br />
+* Develop your feature on Xcode IDE  and run it .<br />
```

Comment:
This is very Mac specific, and does not apply to most LuaRocks users/developers. Did you copy this from somewhere else?